src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationRejection.scala

```diff
@@ -1,6 +1,29 @@
 package ch.epfl.bluebrain.nexus.admin.organizations
+import java.util.UUID
+
 import ch.epfl.bluebrain.nexus.commons.types.Rejection
 
 sealed trait OrganizationRejection extends Rejection
 
+/**
+  * Signals the the organization already exists.
+  */
 case object OrganizationAlreadyExistsRejection extends OrganizationRejection
+
+/**
+  * Signals that the organization does not exist.
+  */
+case object OrganizationDoesNotExistRejection extends OrganizationRejection
+
+/**
+  * Signals that the provided revision does not match the latest revision
+  * @param latestRev    latest know revision
+  * @param providedRev  provided revision
+  */
+final case class IncorrectOrganizationRevRejection(latestRev: Long, providedRev: Long) extends OrganizationRejection
```

Comment:
please sync the rejections (naming, fields, use etc.) with the ones from projects